css1024.css

```diff
@@ -0,0 +1,131 @@
+*,
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  color: #555;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+  max-width: 1024px;
```

Comment:
- [ ] Is this really necessary? Why don't you just make the body that 100% of the page regardless of it's size in the main CSS file? 
Ex:
`max-width: 100vw`
Try to write styles that require the least amount of changes to be responsive, that will make your CSS files lighter.